docs/vm/backup-restore.md

```diff
@@ -68,3 +68,42 @@ You can choose to either delete or retain the previous volumes. By default, all
 1. Click `Create`.
 
 The restore process can be viewed from the `Virtual Machines` page.
+
+## Restore a new VM on another Harvester cluster
+
+_Available as of v1.0.0_
+
+You can migrate a backup to another Harvester cluster and create a new VM based on it.
+
+**Requirements:** The VM image name and content must be the same as configured in the cluster where the backup was migrated from.
+
+### Upload the same VM images to a new cluster
+
+1. Get the VM image names in the current cluster.
+```
+> kubectl get vmimage -o json | jq '.items[].metadata.name'
+"image-dgf27"
+```
+2. Apply a VM image YAML with the same name and content in the new cluster.
+```
+> cat <<EOF | kubectl apply -f -
+apiVersion: harvesterhci.io/v1beta1
+kind: VirtualMachineImage
+metadata:
+  name: image-dgf27
+  namespace: default
+spec:
+  displayName: opensuse-leap-15.3.x86_64-nocloud.qcow2
+  pvcName: ""
+  pvcNamespace: ""
+  sourceType: download
+  url: http://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.3/images/openSUSE-Leap-15.3.x86_64-NoCloud.qcow2
+EOF
+```
+
+### Restore a new VM in a new cluster
+
+1. Setup the same backup target in a new cluster.
+2. Go to the `Backups` page.
+3. Specify the new VM name and click `Create`.
```

Comment:
3. Select the synced VM backup metadata and choose to restore a new VM with a specified VM name.

Reply:
Updated it. Thank you.